www/lobby/index.ts

```diff
@@ -0,0 +1,44 @@
+import { Router, Request, Response } from 'express';
+import { Lobby } from '../../lib/lobby';
+import { User } from '../../lib/user';
+import { validateJwt } from '../../utils/jwt';
+import { lobby_id_router } from './:id';
+
+export const lobby_router = Router();
+
+lobby_router.use('/middleware', validateJwt);
+
+// Creates a new lobby and returns the lobby id w/ lobby data
+/**
+ * Body Params: lobbyName, theme, id, refreshToken
+ */
+lobby_router.post('/', async (req: Request, res: Response) => {
+  const lobbyName = req.body.lobbyName;
+  const theme = req.body.theme;
+  const userId = req.body.id;
+
+  const manager = await User.fromId(userId);
```

Comment:
i think our user class wil have to track what aurgies they have created at some point.. might be good to make an issue to detail how to do that!